CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+1.18.0 / 2019-11-08
```

Comment:
I appreciate I'm not in the loop on timings for deployment but I'm a bit surprised this is being released today. If it isn't can the date just be set to `TBC` please?